gnomad/resources/config.py

```diff
@@ -13,17 +14,28 @@ class GnomadPublicResourceSource(Enum):
     AZURE_OPEN_DATASETS = "Azure Open Datasets"
 
 
-DEFAULT_GNOMAD_PUBLIC_RESOURCE_SOURCE = (
-    GnomadPublicResourceSource.GOOGLE_CLOUD_PUBLIC_DATASETS
-)
+def get_default_public_resource_source() -> Union[GnomadPublicResourceSource, str]:
+    """
+    Get the default source for public gnomAD resources.
+
+    :returns: Default resource source
+    """
+    default_source_from_env = os.getenv("GNOMAD_DEFAULT_PUBLIC_RESOURCE_SOURCE", None)
+    if default_source_from_env:
+        # Convert to a GnomadPublicResourceSource enum if possible
+        try:
+            default_source = GnomadPublicResourceSource(default_source_from_env)
+            return default_source
+        except ValueError:
```

Comment:
Should we throw a warning that the environment variable defines a source that is not one of those in `GnomadPublicResourceSource`?

Reply:
I debated about this. Users can provide their own source, such as `gs://my-bucket/gnomad-resources`, so this is (potentially) valid. I'll add some logging here.